manifests/iop_advisor_engine.pp

```diff
@@ -0,0 +1,64 @@
+# Contains certs specific configurations for advisor
+class certs::iop_advisor_engine (
+  Stdlib::Fqdn $hostname = 'localhost',
```

Comment:
I don't like certificates for localhost. I know we have it for candlepin, but is there a way we can avoid this?

Reply:
I think it's conceptually the same problem as with candlepin: we run the service on localhost only so it's not reachable via the network (without us having to manage a firewall), but that means that the cert if it should validate be for localhost.

Reply:
I guess the only reasonable no-SSL answer are unix sockets, but I am not sure I want to dive into that direction with containers just yet

Reply:
We opted for this route for consistency with our Candlepin deployment model. And to give the option to move off localhost if we ever needed to. I am wondering if we should stick with consistency here, and aim for changing how we handle localhost services in our container deployment.

Reply:
@ekohl would you be OK with this as-is for now, so we have it at least working, and then re-consider when coming closer to a real release?